css-text/Overview.bs

```diff
@@ -755,6 +755,9 @@ Languages and Typesetting</h4>
          the UA must either <a>hang</a> the <a>white space</a>
          or visually collapse their character advance widths
          such that they don't take up space in the line.
+         However, if 'overflow-wrap' is set to ''break-spaces'',
```

Comment:
I think this "but" could just as easily be replaced with "and". We're talking about a subset of of the non-collapsible spaces. "If there are non-collapsible spaces and when 'white-space' is set to ''pre-wrap''" is the same as "if there are non-collapsible spaces, but [only in the case when] 'white-space' set to ''pre-wrap''"